### ISSUE TITLE
Roadmap to version 1.0

### ISSUE THREAD

**Issue by leepeuker** [leepeuker/movary#272]

**Comment by JVT038** [comment]
I personally think that if you want to attract a lot of people to use this, it should be easy for them to migrate from whatever they're using (Netflix, Plex, Jellyfin, Trakt, etc.) to Movary.
So this means adding built-in support for these services and make it easy for them to move.

**Comment by leepeuker** [comment]
I agree that providing support for more third party imports is important to position Movary as a central point between all those services. However I do not see this mandatory for version 1.0. 

These are the requirements I had in mind for the first stable version:
- you can easily manage (create, edit, delete) your movie plays and user data
- you can easily see your latest history and aggregated facts about it
- you can keep Movary in sync with the main third party services it wants to replace/complement: trakt and letterboxd
- you can scrobble plays from the biggest self hosted streaming solutions: plex and jellyfin
- the app looks/feels smooth and streamlined
- you can easily set it up, run it and keep it up to date

Adding more third party support (and maintain it!) will be an ongoing task, but it does not affect the basic functionality of Movary.

**Comment by JVT038** [comment]
Alright.

> the app looks/feels smooth and streamlined

I think that an important part for this, is to replace all `alert()` dialogs with a bootstrap modal. An alert dialog looks ugly in my opinion, and I think a bootstrap modal is visually more appealing.
